profile_library/ikea/LED1739R5/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-11-20T11:28:25.162656",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plug S",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.15.4:docker"
+  },
+  "name": "LED1739R5",
```

Comment:
This needs to be the marketed name, this is shown in the library browser.
Please see the other Ikea lights for example: https://library.powercalc.nl/?manufacturer=ikea
Name column